app/uk/gov/hmrc/fhregistrationfrontend/views/forms/main_business_address.scala.html

```diff
@@ -42,126 +51,177 @@
-
-    @helper.CSRF.formField
-
-    @RadioGroup(
-      RadioGroupParams(
-        name = Some(timeAtCurrentAddressKey),
-        legend = Some(Messages("fh.main_business_address.time_at_address.label")),
-        inline = false,
-        error = Some(Messages("fh.timeAtCurrentAddress.error.required")),
-        hasErrors = mainBusinessAddressForm(timeAtCurrentAddressKey).hasErrors,
-        value = mainBusinessAddressForm(timeAtCurrentAddressKey).value,
-        hintText = Some(Messages("fh.main_business_address.time_at_address.hintText")),
-        radios = List(
-          RadioParams(
-            Messages("fh.main_business_address.0to3years.value"),
-            Messages("fh.main_business_address.0to3years.label"),
-            dataTarget = Some("lessThan3Years")
-          ),
-          RadioParams(
-            Messages("fh.main_business_address.3to5years.value"),
-            Messages("fh.main_business_address.3to5years.label")
-          ),
-          RadioParams(
-            Messages("fh.main_business_address.5to10years.value"),
-            Messages("fh.main_business_address.5to10years.label")
-          ),
-          RadioParams(
-            Messages("fh.main_business_address.10plusYears.value"),
-            Messages("fh.main_business_address.10plusYears.label")
+@lessThan3Year = {
+      @viewHelpers.govUkRadios(Radios(
+          fieldset = Some(Fieldset(
+              legend = Some(Legend(
+                  content = Text(Messages("fh.main_business_address.previous.label")),
+                  classes = "govuk-fieldset__legend--m",
+                  isPageHeading = false
+              ))
+          )),
+          idPrefix = Some(previousAddressKey),
+          name = previousAddressKey,
+          errorMessage = if (mainBusinessAddressForm(previousAddressKey).hasErrors) {
+              Some(ErrorMessage(content = Text(messages("fh.previousAddress.error.required"))))
+          } else None,
+          hint = Some(Hint(
+           content = Text(Messages("fh.main_business_address.previous.hintText"))
+          )),
+          items = Seq(
+              RadioItem(
+                  content = Text(Messages("fh.generic.yes")),
+                  value = Some("true"),
+                  id = Some("previousAddresstrue"),
+                  checked = mainBusinessAddressForm(previousAddressKey).value.toList.contains("true"),
+                  conditionalHtml = Some(manualAddress)
+              ),
+              RadioItem(
+                  content = Text(Messages("fh.generic.no")),
+                  id = Some("previousAddressfalse"),
+                  checked = mainBusinessAddressForm(previousAddressKey).value.toList.contains("false"),
+                  value = Some("false")
+              )
           )
-        )
+      ))
+}
+
+@layout(title = titlePrefix + title, Some(pageScripts)) {
+
+  <link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>
```

Comment:
Additional css files should be passed through via a pageHeadBLock, not added mid-page.
See https://github.com/hmrc/fh-registration-frontend/blob/main/app/uk/gov/hmrc/fhregistrationfrontend/views/acknowledgement_page.scala.html#L32-L34 for example.